Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_test.py

```diff
@@ -1769,3 +1769,84 @@ def test_get_runtime_container_audit_events_command(requests_mock):
     args = {}
 
     assert get_runtime_container_audit_events(client, args).raw_response == response
+
+
+@pytest.mark.parametrize(
+    # Write and define the expected
+    "action, response",
+    [
+        ("archive", "Incident 1111 was successfully archived"),
+        ("unarchive", "Incident 1111 was successfully unarchived")
+    ]
+)
+def test_archive_audit_incident_command(mocker, action, response):
+    """
+    Given:
+        - An app client object
+        - Relevant arguments
```

Comment:
please separate the inputs to cases

Reply:
What do you mean by separating to cases?